go/ethadapter/blockprovider.go

```diff
@@ -58,9 +58,15 @@ func (e *EthBlockProvider) StartStreamingFromHeight(height *big.Int) (*host.Bloc
 	return &host.BlockStream{Stream: streamCh, Stop: cancel}, nil
 }
 
-func (e *EthBlockProvider) IsLive(h gethcommon.Hash) bool {
+func (e *EthBlockProvider) IsLive(b *types.Block) bool {
```

Comment:
This function needs a comment I feel.

Reply:
Perhaps renaming it to `isLatest` would also be good in order to be consistent with the naming inside the enclave. I missed initially that it was the same thing when reviewing

Reply:
Good shout